Client/Frontend/Settings/SyncContentSettingsViewController.swift

```diff
@@ -6,13 +6,120 @@ import Foundation
 import Shared
 import Sync
 
+class ManageSetting: Setting {
+    let profile: Profile
+
+    override var accessoryType: UITableViewCellAccessoryType { return .disclosureIndicator }
+
+    override var accessibilityIdentifier: String? { return "Manage" }
+
+    init(settings: SettingsTableViewController) {
+        self.profile = settings.profile
+
+        super.init(title: NSAttributedString(string: Strings.FxAManageAccount, attributes: [NSForegroundColorAttributeName: SettingsUX.TableViewRowTextColor]))
+    }
+
+    override func onClick(_ navigationController: UINavigationController?) {
+        let viewController = FxAContentViewController(profile: profile)
+
+        if let account = profile.getAccount() {
+            var cs = URLComponents(url: account.configuration.settingsURL, resolvingAgainstBaseURL: false)
+            cs?.queryItems?.append(URLQueryItem(name: "email", value: account.email))
+            if let url = try? cs?.asURL() {
```

Comment:
If this doesn't work, is the view controller just going to be blank? What happens?

Reply:
This shows the FxA login page, I think this is a decent fallback for now.

Reply:
It'll use the profile's `signInURL`, right? I wonder what happens if you're trying to sign in to a custom server…